azure-pipelines.yml

```diff
@@ -0,0 +1,14 @@
+# ASP.NET Core
+# Build and test ASP.NET Core projects targeting .NET Core.
+# Add steps that run tests, create a NuGet package, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core
+
+pool:
+  vmImage: 'Ubuntu 16.04'
```

Comment:
Not sure if we have a choice, but do we actually want to test on Linux? I'm not sure we're planning to develop or deploy on Linux

Reply:
I can see what the other templates have as options

Reply:
CC @asyasky

Reply:
Fixed (also, FYI, I'm the only one who can sign in to the admin account or do CI stuff right now, so CCing me was unnecessary :) )